src/main/java/com/google/sps/servlets/FetchForumServlet.java

```diff
@@ -47,14 +47,14 @@
   // This is the query that will be executed.
   String query;
```

Comment:
I don't think there is a need to declare this beforehand. You could do it like you did it in the PostQuestionServlet.java file, where you declare it where you need to instantiate it (line 58).

Reply:
done